sdk/state/open.go

```diff
@@ -168,7 +184,7 @@ func (c *Channel) ConfirmOpen(m Open) (open Open, fullySigned bool, err error) {
 	fullySigned = true
 	c.latestCloseAgreement = CloseAgreement{
 		IterationNumber:       1,
-		Balance:               Amount{},
+		Balance:               Amount{Asset: m.Asset},
```

Comment:
the balance on the stored `latestClosAgreement` now stores the asset type and doesn't assume native